inst/pages/30_differential_abundance.qmd

```diff
@@ -568,6 +568,32 @@ zicoseq_res %>%
   knitr::kable()
 ```
 
+## Visualizing DAA
+
+In order to visualize results in a differential abundance analysis, we can use the `plotExpression` function from the scater library. This will show a jitterplot with the given results. Here's an example :
+
+```{r visualize DAA}
+library(scater)
+
+#Let's store the top three taxa
+
+top_taxa <- ancombc2_out$res %>%
+  dplyr::select(starts_with("taxon")) %>%
+  head(3) %>%
+  unlist()
+
+#Creates jitter plots for the top three taxa per patient status and colored by cohort
+plotExpression(
+  tse, top_taxa,
+  exprs_values = "relabundance",
```

Comment:
scater::plotExpression is nowadays also using `assay.type` instead of `exprs_values`. This should be changed.